src/client/src/ui/spotTile/model/spotTileModel.js

```diff
@@ -213,4 +223,9 @@ export default class SpotTileModel extends ModelBase {
         }
     }
   }
+
+  _startChartIQTimer(){
+    this.chartIQisOpening = true;
+    this._schedulerService.async.scheduleFuture('', 2000, ()=> this.chartIQisOpening = false);
```

Comment:
this needs to happen on the esp dispatch loop else the view won't know there was a change. something like this:
```
this._schedulerService.async.scheduleFuture('', 2000, ()=> { 
  this.router.runAction(theModelId, () => {
    this.chartIQisOpening = false; 
  }
});
```
or preferably (and handling the disposable):

```
// note syntax might be off, just coded it from memory
this.addDisposable( 
  Observable.timer(2000, this._schedulerService.async)
    .subscribeWithRouter(
      this.router, 
      this.modelId, 
      ()=> {
        this.chartIQisOpening = false
      }
);
```
[more info](https://github.com/Reactive-Extensions/RxJS/blob/master/doc/api/core/operators/timer.md)